czmake/utils.py

```diff
@@ -44,18 +59,21 @@ def parse_option(s):
         key = s[:colon]
         ty = s[colon + 1:eq].lower()
         if ty == 'BOOL':
-            value = str2bool(s[eq+1])
+            value = str2bool(s[eq + 1])
         else:
-            value = s[eq+1]
+            value = s[eq + 1]
     return key, value
 
+
 def mkdir(path):
+    # TODO: can we directly use os.makedirs?
```

Comment:
No, if you want to support Python 2, the problem is that `os.makedirs` raised an OS error if the path already existed, in Python 3 they added the `exist_ok` keyword argument that removes this limitation